core/edit/edit.talon

```diff
@@ -125,12 +125,18 @@ new line above: edit.line_insert_up()
 new line below | slap: edit.line_insert_down()
 
 # Insert padding with optional symbols
-(pad | padding): user.insert_between(" ", " ")
+(padding): user.insert_between(" ", " ")
 (pad | padding) <user.symbol_key>+:
     insert(" ")
     user.insert_many(symbol_key_list)
     insert(" ")
 
+# Insert delimiter pairs
+<user.delimiter_pair>: user.delimiter_pair_insert(delimiter_pair)
```

Comment:
It seems slightly weird to define these in edit vs symbols.talon. I personally don't really care, but adds another place to find symbol-y things.

Reply:
moved